modules/nf-scil/utils/extractb0/environment.yml

```diff
@@ -0,0 +1,6 @@
+name: "utils_extractb0"
+channels:
+  - Docker
+  - Apptainer
+dependencies:
+  - "Scilpy"
```

Comment:
Seems like we need to change `Scilpy` to `scilpy` 😄 (you can leave it as is, and we can change it once we go through other modules).